examples/rollup.config.js

```diff
@@ -218,23 +218,25 @@ const targets = [
 // We skip building PlayCanvas ourselves when ENGINE_PATH is given.
 // In that case we have a watcher which copies all necessary files.
 if (ENGINE_PATH === '') {
-    /** @type {RollupOptions|undefined} */
-    let target;
+    const pushTarget = (...args) => {
+        targets.push(buildTarget(...args));
+    };
```

Comment:
```suggestion
    /** @type {buildTarget} */
    const pushTarget = (...args) => {
        targets.push(buildTarget(...args));
    };
```

(lets keep the type for full typing / IntelliSense)